README.md

```diff
@@ -9,7 +9,7 @@ This node module provides an adapter to the PostgreSQL database.
 
 This is a first alpha version! It can connect to a PostgreSQL database and execute statements, currently `SELECT` for entitysets and entites only.
 
-Please see [`CONTRIBUTING.md`](./docs/CONTRIBUTING.md) for how to constribute additional capabilites!
+Please see [`CONTRIBUTING.md`](./docs/CONTRIBUTING.md) for how to contribute additional capabilites!
```

Comment:
thx for spotting that :)
would you mind fixing capabilit**i**es as well while you're on it?

Reply:
done